low-level/sensors/impl/centos_7/service_monitor.py

```diff
@@ -169,22 +173,31 @@ def __init__(self, unit):
         self.previous_substate = "N/A"
         self.previous_pid = "N/A"
         self.nonactive_enter_timestamp = time.time()
+        self.active_enter_timestamp = time.time()
         self.nonactive_threshold = int(
             Conf.get(SSPL_CONF,
                      f"{ServiceMonitor.name().upper()}>threshold_inactive_time",
                      '60'))
+        self.active_threshold = int(Conf.get(SSPL_CONF,
+                     f"{ServiceMonitor.name().upper()}>threshold_active_time",
+                     '30'))
         self.properties_changed_signal = None
         self._service_state = ActiveState
         self._unit_state = None
 
     @property
     def is_enabled(self):
-        return 'disabled' not in self.properties_iface.Get(
-            UNIT_IFACE, 'UnitFileState')
+        if self.properties_iface.Get(UNIT_IFACE, 'UnitFileState'):
```

Comment:
Did @sandeepsgt confirmed if this is fine. Also i had some comment around this i believe

Reply:
Yes, He himself only suggested above change. [comment](https://github.com/Seagate/cortx-monitor/pull/444#discussion_r666803104) @sandeepsgt  please confirm.
I have tested it, with tis change we raise alert for services which are configured on system.

Reply:
Got now after our discussion @shriya-deshmukh 
Only if service file is present, then check if disabled in that service file & return true/false accordingly , else return false.